homeassistant/components/deconz/__init__.py

```diff
@@ -58,28 +57,25 @@
     return True
 
 
-async def async_setup_entry(hass, entry):
+async def async_setup_entry(hass, config_entry):
     """Set up a deCONZ bridge for a config entry."""
     if DOMAIN in hass.data:
         _LOGGER.error(
             "Config entry failed since one deCONZ instance already exists")
         return False
-    result = await async_setup_deconz(hass, None, entry.data)
-    if result:
-        return True
-    return False
+    return await async_setup_deconz(hass, config_entry)
 
 
-async def async_setup_deconz(hass, config, deconz_config):
+async def async_setup_deconz(hass, config_entry):
```

Comment:
Any reason this is a standalone method and not just part of `async_setup_entry` ?

Reply:
Only legacy reasons, which are now irrelevant. Thanks for pointing that out!